evergreen/evergreen.yml

```diff
@@ -239,9 +239,9 @@ functions:
           export FLE_AWS_ACCESS_KEY_ID=${FLE_AWS_ACCESS_KEY_ID}
           export FLE_AWS_SECRET_ACCESS_KEY=${FLE_AWS_SECRET_ACCESS_KEY}
           ${PREPARE_SHELL}
-          evergreen/add-certs.sh
+          SSL=${SSL} evergreen/add-certs-if-needed.sh
```

Comment:
maybe I missed something, why don't we use `export` as in `FLE_AWS_SECRET_ACCESS_KEY`?

Reply:
You're right we could also do it that way I believe :) We export the FLE keys before calling `{PREPARE_SHELL}` so that we don't leak the keys in the logs, so in this case, we could export after `PREPARE_SHELL`.  In the end, I chose to follow the precedent set by run-tests.sh. :)